docs/open-source-projects.mdx

```diff
@@ -219,13 +219,13 @@ export const projectsData = [
 			<tr>
 			<td>
 				<a href="https://mainnet.eosio.online">
-					<img src="https://eosio.online/images/eos.png" width="30%"/>
+					<img src="../img/eos.jpg" width="30%"/>
```

Comment:
you could also try setting the root as an absolute path, for example: 

`/img/eos.jpg` should work from any location as it references the root dir